src/plugin/fieldViews/NestedElementFieldView.ts

```diff
@@ -82,9 +87,10 @@ export class NestedElementFieldView extends ProseMirrorFieldView {
     offset: number,
     // The initial decorations for the FieldView.
     decorations: DecorationSource,
-    { placeholder, isResizeable }: NestedElementFieldDescription,
+    { placeholder, isResizeable, minRows }: NestedElementFieldDescription,
     // Specify plugins of which the field should have its own copy
     allowedPlugins: PluginKey[] = []
+    // Initial height of the field, in rows
```

Comment:
Not sure if this comment should be here?

Reply:
Definitely not, thanks for spotting!